spring-batch-docs/src/main/asciidoc/domain.adoc

```diff
@@ -77,7 +77,6 @@ public Job footballJob() {
                      .start(playerLoad())
                      .next(gameLoad())
                      .next(playerSummarization())
-                     .end()
```

Comment:
This can be used if `playerLoad()` is flow. But it seems that `playerLoad()` is step [according to context](https://github.com/spring-projects/spring-batch/blob/main/spring-batch-docs/src/main/asciidoc/step.adoc#step-restart-configuration-example).